config/routes.rb

```diff
@@ -454,13 +451,13 @@
     resources :work
 
     # legacy
-    get :chat, to: redirect('/chat/%{product_id}')
```

Comment:
Line is too long. [97/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.